module/Swissbib/src/Swissbib/Controller/DetailPageController.php

```diff
@@ -131,7 +132,7 @@ protected function getSubjectsOf(array $subjectIds): array
      *
      * @return array
      */
```

Comment:
Renamed this method because it provides multiple medias and not only a single one

Reply:
Agreed